go/vumitools/middleware.py

```diff
@@ -631,10 +632,18 @@ def setup_middleware(self):
         self.redis_manager = yield TxRedisManager.from_config(
             self.config.redis_manager)
         self.redis = self.redis_manager.sub_manager(self.SUBMANAGER_PREFIX)
+        self.local_recent_convs = set()
+        self._looper = LoopingCall(self.reset_local_recent_convs)
+        self._looper.start(1800)
```

Comment:
We should perhaps make this configurable? 1800 seems like a good default value though.